README.md

```diff
@@ -2,26 +2,54 @@
 
 ![Screenshot from 2023-03-18 15-29-05](https://user-images.githubusercontent.com/8228060/226112255-fe2d4cdc-193e-4c23-8a36-3d8f60baaf03.png)
 
+This project contains an awesome application called `QuickPizza` and also demonstrates how to use the different features of k6. 
```

Comment:
In the hackathon project, I used QuickPizza; I listed a few technical details of the app:

![Screenshot from 2023-06-22 13-41-34](https://github.com/grafana/quickpizza/assets/8228060/bb0b4ee9-4f1d-4989-a8e4-0fc176bac321)

It would be lovely if we could provide some of that info!

Reply:
I've added a few details! Please look at https://github.com/grafana/quickpizza/pull/4/commits/8dc9eb68e0fdb3d1f6424de3ccd08f25113baa18#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5-R9